### PR TITLE
Fix: Fixed an issue where the Layout Picker UI is broken up in a certain localization

### LINKED ISSUE #14880: Bug: Layout flyout UI may be inconsistent due to localization
### Description

The `Layout` flyout (opened from the toolbar button) may have an inconsistent UI due to text localization.
In my case, I have to scroll to access switches

### Steps To Reproduce

1. Switch language to `Italian`
2. Open the `Layout` flyout
3. Set `Riquadri` view (`Tiles`)
4. See that the layout is messed up by the warning

### Requirements

- Set text wrap in the `Layout` flyout (`Size` section)

### Files Version

3.2.5.0

### Windows Version

10.0.22631.3155

### Log File

image

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

As the title.

## PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #14880
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Change language to Italian
   2. Go to a folder in a Tiles layout
   3. Open the Layout Picker

## Screenshots

Before|After
---|---
image|image

### CODE REVIEW

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 1]
```diff
@@ -644,7 +644,7 @@
 
 				<AppBarButton.Flyout>
 					<Flyout contract8Present:ShouldConstrainToRootBounds="False" Placement="Bottom">
-						<StackPanel Spacing="12">
+						<StackPanel Width="392" Spacing="12">
```
Comment (0x5bfa) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:647]: This is surely needed to set the maximum size of the Flyout.

Comment (0x5bfa) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:647]: Alright then. If so, do we want to conteralize the TextBlock?

Comment (yaira2) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:647]: We can set a max width

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**